README.md

```diff
@@ -1,16 +1,14 @@
 # XRP Ledger Standards / Drafts
 
-To ensure interoperability between XRP (Ledger) applications, tools, platforms, pursuing a great user experience, it would be best if the
-community (developers, users) agree on certain implementations.
+To ensure interoperability between XRP (Ledger) applications, tools, platforms, pursuing a great user experience, the community (developers, users, etc.) _should_ agree on certain specifications.
   
-# Contributing / Workflow
+# [Contributing](./CONTRIBUTING.md)
 
-### All drafts start as [**DISCUSSIONS**](https://github.com/xrp-community/standards-drafts/discussions).
-
-Drafts (discussions) are considered work in progress, and are up for [discussion (please discuss, ask, suggest, add, ...)](https://github.com/xrp-community/standards-drafts/discussions). Once settled on the standard, the discussion will be converted to an **ISSUE**, to be referred to in a **PULL REQUEST** adding the standard to the **CODE** (commit). Now the issue can be resolved.
+Please see [CONTRIBUTING.md](./CONTRIBUTING.md)
 
 # Directory
-When a standard moves to a folder + file(s) in the Code section of this repository, it will be added to the `standards.toml` file:
+
```

Comment:
Would a folder be created once the Standards draft is published as a Candidate Specification?

Reply:
No; each Draft should have a folder as well. [XLS-2d](https://github.com/xpring-eng/XRPL-Standards/tree/master/XLS-2d) is an example of a Draft that has a folder.